CLAUDE.md

```
@@ -131,39 +147,36 @@ zsh 関数 (`gg`, `mkcd`, `ccd`, `ghq` ラッパ) と機能パリティを保つ
 
 ### Neovim Structure (`dot_config/nvim/`)
 
+`features.neovim = true` のときのみ展開される。
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)